docs/troubleshooting/15_magic-link-verification-url.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: magic-link-verification-url
+title: Magic links use old custom domain name
+sidebar_label: Wrong domain in magic links
```

Comment:
```suggestion
sidebar_label: Wrong domain in verification links
```

If I understand correctly this only applies to verification links? 

I am not super happy with the term Magic Links in the sidebar, as it is used to refer to  recovery links as well:
https://ory.sh/docs/kratos/manage-identities/account-recovery#magic-links

If this also affects recovery we should mention it.

As an aside: Maybe "one-time login links" would also be a good alternative to "magic links", bc I honestly dont know what "magic" means 
(It is fairly widespread so I know the term, but if you never came across it idk)

Reply:
I'd stick to using the term "magic links" as it's commonly used in both our docs and the console. Introducing another name will confuse the users I think. 

And yes, the user reported that he encountered the problem with verification links, but even that error wasn't really reliably reproducible. This leads me to think that it can probably happen to any flow that uses this method so I'll just leave it as is - "flow-agnostic". ;)